Examples/Python/src/Geometry.cpp

```diff
@@ -294,16 +296,15 @@ void addGeometry(Context& ctx) {
   {
     auto cylStack = py::class_<CylinderVolumeStack>(m, "CylinderVolumeStack");
 
-    py::enum_<CylinderVolumeStack::AttachmentStrategy>(cylStack,
-                                                       "AttachmentStrategy")
-        .value("Gap", CylinderVolumeStack::AttachmentStrategy::Gap)
-        .value("First", CylinderVolumeStack::AttachmentStrategy::First)
-        .value("Second", CylinderVolumeStack::AttachmentStrategy::Second)
-        .value("Midpoint", CylinderVolumeStack::AttachmentStrategy::Midpoint);
+    py::enum_<VolumeAttachmentStrategy>(cylStack, "AttachmentStrategy")
+        .value("Gap", VolumeAttachmentStrategy::Gap)
+        .value("First", VolumeAttachmentStrategy::First)
+        .value("Second", VolumeAttachmentStrategy::Second)
+        .value("Midpoint", VolumeAttachmentStrategy::Midpoint);
 
-    py::enum_<CylinderVolumeStack::ResizeStrategy>(cylStack, "ResizeStrategy")
-        .value("Gap", CylinderVolumeStack::ResizeStrategy::Gap)
-        .value("Expand", CylinderVolumeStack::ResizeStrategy::Expand);
+    py::enum_<VolumeResizeStrategy>(cylStack, "ResizeStrategy")
+        .value("Gap", VolumeResizeStrategy::Gap)
+        .value("Expand", VolumeResizeStrategy::Expand);
   }
```

Comment:
I think they should also not be nested in the `CylinderVolumeStack` class on the python side, but rather be top-level members of the `acts` package.